src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/Et1VettingControllerTest.java

```diff
@@ -74,14 +93,51 @@ void initAddJudge_invalidToken() throws Exception {
     }
 
     @Test
-    void initAddJudge_BadRequest() throws Exception {
+    void initialiseEt1Vetting_BadRequest() throws Exception {
         when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(true);
-        mockMvc.perform(post(INIT_CASE_VETTING)
+        mockMvc.perform(post(INIT_CASE_VETTING_ENDPOINT)
                         .contentType(APPLICATION_JSON)
                         .header(HttpHeaders.AUTHORIZATION, AUTH_TOKEN)
                         .content("error"))
                 .andExpect(status().isBadRequest());
         verify(et1VettingService, never()).initialiseEt1Vetting(ccdRequest.getCaseDetails());
     }
 
+    @Test
+    void jurisdictionCodes_Success() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(true);
+        when(et1VettingService.validateJurisdictionCodes(any())).thenReturn(Arrays.asList());
+        when(et1VettingService.populateEt1TrackAllocationHtml(any())).thenReturn("jurCodeHtml");
+        mockMvc.perform(post(JURISDICTION_CODE_ENDPOINT)
+                .contentType(APPLICATION_JSON)
+                .header(HttpHeaders.AUTHORIZATION, AUTH_TOKEN)
+                .content(jsonMapper.toJson(ccdRequest)))
+            .andExpect(status().isOk())
+            .andExpect(jsonPath("$.data.trackAllocation", notNullValue()))
+            .andExpect(jsonPath("$.errors", empty()))
+            .andExpect(jsonPath("$.warnings", nullValue()));
+        verify(et1VettingService, times(1)).validateJurisdictionCodes(any());
+        verify(et1VettingService, times(1)).populateEt1TrackAllocationHtml(any());
+    }
+
+    @Test
+    void jurisdictionCodes_invalidToken() throws Exception {
+        when(verifyTokenService.verifyTokenSignature(AUTH_TOKEN)).thenReturn(false);
+        mockMvc.perform(post(JURISDICTION_CODE_ENDPOINT)
+                .contentType(APPLICATION_JSON)
+                .header(HttpHeaders.AUTHORIZATION, AUTH_TOKEN)
+                .content(jsonMapper.toJson(ccdRequest)))
+            .andExpect(status().isForbidden());
+        verify(et1VettingService, never()).initialiseEt1Vetting(ccdRequest.getCaseDetails());
+    }
```

Comment:
Do these need to throw Exception?
Only throw if we really need to.  Also better to throw the specific types that just Exception as that helps a dev understand what exception could be thrown